routes/store.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const Store = require('../models/store');
+
+const router = express.Router();
+
+const jsonParser = bodyParser.json();
+
+router.post('/', jsonParser, (req, res, next) => {
```

Comment:
Instead of using the `jsonParser` middleware inline here, I would either call `app.use(…)` in `index.js` or apply it to the router using `router.use(…)`